### DISCUSSION TITLE
Example Manufacturing Fusion 360 File

### DISCUSSION THREAD

**Discussion by kraskat** [EvilHacker/BoxJoint#1] [General]
Hello, I just came across this plug-in. Honestly it is amazing. Thanks for making it open-source.

I am relatively new to CNC/Fusion 360 and I was wondering, if you would mind sharing a Fusion 360 file with the manufacturing steps for the box joint. I am mainly wondering what operations I should use for the box joint to get it done on my onefinity elite CNC. I saw that you already did it as part of the video on autodesk. Would love to take a look at the file. Thanks in advance.

**Comment by EvilHacker** [comment]
Here is an example model including the Manufacturing setup.
[Box Cutting.f3d.zip](https://github.com/EvilHacker/BoxJoint/files/14392848/Box.Cutting.f3d.zip)

I cut out this box using three 3D Contour Milling operations with one flat end mill.

1)
The first operation only cuts down to the level of the flat part of the dogbone. This will be 1/2 of the "Tool Diameter" parameter of the joint. Depending on your machine and the material you are cutting, this can be done in one or two passes.

Important settings in the 3D Contour Milling operation:
```
  Heights:
    Bottom Height
      From: Stock top
      Offset: -toolDiameter / 2
```

2)
The second operation is a bottom-up cut and bi-directional. It takes small step following the curvature of the fingers from the bottom to the top. There is relatively little material being removed so the cutting can be done in both directions and at higher speed.

Important settings in the 3D Contour Milling operation:
```
  Heights:
   Bottom Height
     From: Stock top
     Offset: -toolDiameter / 2
  Passes:
    Maximum Stepdown: [very small]
    Direction: Both ways
    Order Bottom-Up: Check
  Linking
    Retraction Policy: Minimum retraction
```

3)
The third operation finishes the cut out from below the curved fingers.

Important settings in the 3D Contour Milling operation:
```
  Rest Machining
    Source: From previous operations(s)
  Heights:
    Bottom Height
      From: Stock bottom
      Offset: 0
```

I find that I have to fiddle with many settings in the Milling operation to get it just right, so there may be some other settings I missed.

Good luck with you project

![cutting](https://github.com/EvilHacker/BoxJoint/assets/6175398/a9872405-4035-4769-8081-1613b1109de6)

**Comment by EvilHacker** [comment]
Here is a video of a simulated manufacturing operation.
[![CAM operation](https://img.youtube.com/vi/0Qt5S2ueKRE/maxresdefault.jpg)](https://youtu.be/0Qt5S2ueKRE)

**Reply by deonholt** [reply]
Do you mind making a video of actually using the app to box joint in Fusion360. I have tried now for 30 minutes and I cannot get it to work,
Please?!

**Comment by jamezz42** [comment]
GREAT APP!!!
Thanks for all the work you put into it. I have a feature request if you are planning any updates, add a clearance option so we can eliminate some sanding to fit.  Thanks again.

**Comment by jamezz42** [comment]
![test box](https://github.com/EvilHacker/BoxJoint/assets/16006715/9be2005a-8dce-432a-8e83-b7bd515e228f)

**Reply by deonholt** [reply]
Do you mind sharing a video on how you actually did it in Fusion360. I have two equal bodies and they touch and I cannot get them to box join.
Please!?
